cosmos/operators/docker.py

```diff
@@ -42,8 +43,10 @@ class DbtDockerBaseOperator(AbstractDbtBaseOperator, DockerOperator):  # type: i
     def __init__(
         self,
         image: str,  # Make image a required argument since it's required by DockerOperator
+        profile_config: ProfileConfig | None = None,
         **kwargs: Any,
     ) -> None:
+        self.profile_config = profile_config
```

Comment:
Since this only is tested with `profiles.yml` and may not work with `ProfileMapping` since the the Airflow connection will not be available in the docker container can we raise a CosmosValueError error if the user providers a `profile_config` that DOES NOT have a [profile_config.profiles_yml_filepath](https://github.com/astronomer/astronomer-cosmos/blob/0e826b1db82969c2555b85d4c4728e7a8373ed58/cosmos/config.py#L275) or if they specify [profile_config.profile_mapping](https://github.com/astronomer/astronomer-cosmos/blob/0e826b1db82969c2555b85d4c4728e7a8373ed58/cosmos/config.py#L278) (which won't work) so we alert them upfront?